package.json

```diff
@@ -78,6 +79,8 @@
     "lint": "prettier --list-different '**/*.{js,css,md,json,yaml,yml}'",
     "format": "prettier --write '**/*.{js,css,md,json,yaml,yml}'",
     "test:ci": "npm run lint -s && npm run dist -s && flow && npm run test:coverage -s -- --outputFile build/test-results.json --json",
-    "prepublishOnly": "npm run dist -s"
+    "prepublishOnly": "npm run dist -s",
+    "docs:generate": "./node_modules/.bin/documentation build src/** -f html -o docs",
```

Comment:
This probably shouldn't point at `src/**`, only at `src/lib/**`, since we don't want docs generated for the demo site. Could this do Markdown instead of HTML?

Reply:
Sure, I'll update the read path to `libs`
It can output `md`, I'll add a command for that.